src/backend/settings/__init__.py

```diff
@@ -153,6 +147,11 @@
 ]
 
 TEMPLATES = [
+    {
+        "BACKEND": "django.template.backends.jinja2.Jinja2",  # TODO: This is terrible
```

Comment:
We only have a few templates, I don't think switching them to django templating would be a huge effort if anyone feels like doing it

Reply:
Django templating does have macros, which we depend on quite heavily. It's doable but I felt it was beyond the scope of this